pvt/benchmarks/src/PoolBenchmark.behavior.ts

```diff
@@ -29,25 +25,59 @@ import {
 import { BaseContract } from 'ethers';
 import { IERC20 } from '@balancer-labs/v3-interfaces/typechain-types';
 
+export enum PoolTag {
+  Standard = 'Standard',
+  WithRate = 'WithRate',
+  ERC4626 = 'ERC4626',
+  WithNestedPool = 'WithNestedPool',
+}
+
+export type PoolInfo = {
+  pool: BaseContract;
+  poolTokens: string[];
+};
+
+export type TestsSwapHooks = {
+  gasTag?: PoolTag;
+  actionAfterFirstTx?: () => Promise<void>;
+};
+
+export type TestsAddLiquidityHooks = {
+  actionAfterFirstTx?: () => Promise<void>;
+};
+
+export type TestSettings = {
+  offNestedPoolTests: boolean;
+};
+
 export class Benchmark {
   _testDirname: string;
   _poolType: string;
+  _settings: TestSettings = { offNestedPoolTests: false };
```

Comment:
Just a little confused about the. naming here. It skips the nested pool tests if this is false, so it means "turn off nested pool tests?" As is, "off" as a prefix like that means something like "based upon" NestedPoolTests, like you're extending them, vs. turning them off.

Maybe call it `withNestedPool` for consistency with the PoolTag?